docs/examples/edb/edb_example.ipynb

```diff
@@ -463,7 +463,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.6"
+   "version": "3.8.12"
```

Comment:
this is fine, but as an aside I wish we didn't have this sort of thing show up as a "change"

Reply:
Noted.